gapic-generator/templates/default/service/test/method/_setup.erb

```diff
@@ -0,0 +1,39 @@
+def setup
+  @test_channel = GRPC::Core::Channel.new "localhost:8888", nil, :this_channel_is_insecure
+  @mock_stub = MiniTest::Mock.new
+  @mock_page_enum = :mock_page_enum
+  @response = :mock_response
+  @grpc_operation = :mock_grpc_operation
+  @options = {}
+end
+
+def with_stubs client: nil, paged: nil
+  client ||= @mock_stub
+  paged ||= @mock_page_enum
+  Gapic::ServiceStub.stub :new, client do
+    Gapic::PagedEnumerable.stub :new, paged do
```

Comment:
I think it would be good to list the arguments, as an added check that the method is being called correctly.

Reply:
For sure, this was something I hadn't gotten to in the first pass. We should add it.

Reply:
This would be another reason to unfactor `with_stubs` back to the test methods.

Reply:
Maybe. Let's get it to work for all the various tests that we'd like and then we can shuffle things around as needed.